packages/cli/src/lib/defaults/infra-modules/http/docker-compose.yaml

```diff
@@ -0,0 +1,8 @@
+version: '3'
+services:
+  http-server:
+    build: ./server
+    environment:
+      PORT: '${PORT:-3500}'
```

Comment:
Should this env var name be more specific and less generic? I fear the name `PORT` could easily conflict with other infra modules that get composed together. Or are env vars isolated to a specific "service" within the docker-compose, and we can indeed have multiple infra modules with the same env var name?